pom.xml

```diff
@@ -187,7 +193,7 @@
             <artifactId>jdi-light-html</artifactId>
             <version>1.4.4</version>
```

Comment:
Some versions are written in the properties, but some are not. It's better to chose only one way.